lib/scenic/configuration.rb

```diff
@@ -0,0 +1,34 @@
+module Scenic
+  class Configuration
```

Comment:
Hmm. I forgot about that being a thing. Do we want to drop 4.1 for it? Is it being at least somewhat widely adopted?